### ISSUE TITLE
esdoc-site

### ISSUE THREAD

**Issue by fkm** [esdoc-next/esdoc-next#5]
After going over the esdoc repositories I wonder, if we should adopt [esdoc-site](https://github.com/esdoc/esdoc-site).

As esdoc is a kind of site generator I don't see the need to create a separate website. But I didn't put much though into this. So, it's possible I overlooked some advantage of the website.

But, my suggestion would be to publish the documentation as generated by esdoc through GitHub Pages.

**Comment by DanielRuf** [comment]
This is already the case there:

https://github.com/esdoc/esdoc-site/blob/master/docs/CNAME

It uses the docs folder.
I think we should fork and continue it. The website (online docs) should not live in the main project.

See how we do it for verdaccio:
https://github.com/verdaccio/website

For me this looks fine. This makes it also easier to move it to some monorepo structure later combined with semantic (rolling) releases.

**Comment by fkm** [comment]
Maybe I should rephrase. At least one part.

What I meant was that esdoc has a website that is different from the documentation esdoc itself generates. And my question is if that is needed or if we could just use the documentation as generated by esdoc as the main website.

The other point about the docs not being part of the main repository is something I can accept. I personally don't see it as a problem and don't mind having a `docs` folder in the project root where I can find help without visiting a website. But that part wouldn't bother me :smiley:

**Comment by DanielRuf** [comment]
We will check what makes sense. In the ideal case we can use esdoc in the esdoc-site repo to copy / generate the needed docs.

**Comment by fkm** [comment]
Yeah. I was wondering if there were any known shortcomings with the generated docs as the original author decided to go with something different. But I take this as a no :sunglasses:
